super_csv/csv_processor.py

```diff
@@ -154,14 +154,14 @@ def get_iterator(self, rows=None, columns=None, error_data=False):
         """
         Export the CSV as an iterator.
         """
+        columns = columns or self.columns
         if error_data:
             # return an iterator of the original data with an added error column
-            rows = self.result_data
-            columns = self.columns + ['status', 'error']
+            rows = rows or self.result_data
+            columns = columns + ['status', 'error']
```

Comment:
would this not be a breaking change?

Reply:
This just respects the ability to override when rows/columns are explicitly provided as kwargs. I don't believe it will be breaking.

Reply:
(i totally missed line 157 when I read this. you're right, no breaking here)